fdbserver/IDiskQueue.h

```diff
@@ -25,6 +25,11 @@
 #include "fdbclient/FDBTypes.h"
 #include "fdbserver/IKeyValueStore.h"
 
+enum class CheckHashes {
```

Comment:
Why not just use a `bool` ?

Reply:
It's personal preference.  I really dislike functions that look like `doFoo(true, true, false, false, true)`, because it always requires going to look at the declaration to see what each of the trues and falses mean, rather than just giving them a name that's meaningful at the callsite.